<launch>
    <arg name="file" default="$(find leo_example_line_follower)/cfg/color_mask.yaml"/>>

    <node name="color_mask_finder" pkg="leo_example_line_follower" type="color_mask.py" clear_params="true" output="screen">
        <rosparam command="load" file="$(arg file)"/>
    </node>

</launch>